.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - task/CSI-2892_cert_community_operator_hub
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image: "csiblock1/ibm-block-csi-operator-amd64:new"
+
+  prepare_env:
+   name: "prepare_env"
+   needs: operator_image_build
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip yq
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: prepare_env
+        id: environment_setup
+        env:
+          operator_image: ${{ needs.operator_image_build.outputs.operator_image }}
+        run: |
+          build/ci/prepare_env.sh
+   outputs:
+     current_operator_image: "${{ steps.environment_setup.outputs.current_operator_image }}"
+     wanted_operator_image: "${{ steps.environment_setup.outputs.wanted_operator_image }}"
```

Comment:
if I understand them correctly, consider renaming to something like `operator_image_in_branch` and `operator_image_for_test` respectively